modules/blocks.php

```diff
@@ -35,3 +35,8 @@ function jetpack_map_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'map', $dependencies );
 	return preg_replace( '/<div /', '<div data-api-key="'. esc_attr( $api_key ) .'" ', $content, 1 );
 }
+
+jetpack_register_block( 'contact-info' );
+jetpack_register_block( 'email' );
+jetpack_register_block( 'address' );
+jetpack_register_block( 'phone' );
```

Comment:
Since the `email`, `address`, and `phone` blocks are child blocks of `contact-info` -- could you specify their `parent`? We need that for availability to work correctly :grimacing: See #11102 for an example.

Reply:
👍 	Done in #11216